src/App.js

```diff
@@ -1,11 +1,17 @@
 import logo from './logo.svg';
 import './App.css';
-
+import SearchForm from './SearchForm/SearchForm';
+import Counter from './Counter/Counter';
+import GenreSelect from './GenreSelect/GenreSelect';
+import "bootstrap/dist/css/bootstrap.min.css";
 
 function App() {
+  const geners=['Action', 'Comedy', 'Drama', 'Fantasy'];
   return (
     <div className="App">
-   
+     <Counter/>
+     <SearchForm/>
```

Comment:
please, take a look at the task, we need to provide some props to these components, e.g. 

`<SearchForm onSearch={handleSearch} initialValue={initialValue}/>`

Reply:
fixed